index.js

```diff
@@ -12,6 +12,7 @@ function listToArray (list) {
 Toolkit.run(async tools => {
   const template = tools.inputs.filename || '.github/ISSUE_TEMPLATE.md'
   const assignees = tools.inputs.assignees
+  const updateExisting = tools.inputs.update_existing === 'true'
```

Comment:
```suggestion
  const updateExisting = Boolean(tools.inputs.update_existing)
```

The Actions runner treats booleans super weird when parsing the workflow YAML - so this should be a more reliable tweak to make sure that if its present thats enough. We'd also want to remove the `default` key from `action.yml`.